app/assets/javascripts/application.js

```diff
@@ -55,6 +55,110 @@ var ready = function() {
 		}
 	});
 
+	if ('.panel-sort'.length) {
+
+		// Sort panels by stats
+
+		$('.panel-sort').change(function() {
+
+			// get class name of panels and containers
+			var panelClass, panelContainer;
+			if (this.id == "courses") {
+				panelClass = ".course-panel";
+				panelContainer = "#department-courses";
+			} else {
+				panelClass = ".prof-panel";
+				panelContainer = ".prof-panel-container";
```

Comment:
We should maybe use this opportunity to standardize the panel container id. It looks like there isn't an id for the professor container so let's add one while keeping the css class the same.